### DISCUSSION TITLE
Must Have Features Should be added

### DISCUSSION THREAD

**Discussion by Thornvald** [stoatchat/discussions#807] [Feature Suggestions]
List is by priority
Screen Share
Visualization of the members in the voice chat
Adjusting Volume per member in the voice chat
Noise Suppression
Voice Threshold
Sound feedback for buttons
Push to talk

**Comment by jiskayo** [comment]
Can we as a community help you in any way? I mean except for the donations

**Reply by Thornvald** [reply]
I'm not a developer for the Revolt

**Reply by jiskayo** [reply]
> I'm not a developer for the Revolt

I think it has something to do with youe message even if you aren't a developer
